packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -718,7 +638,7 @@ class FlutterPlugin implements Plugin<Project> {
             Task copyFlutterAssetsTask = copyFlutterAssetsTaskProvider.get()
             if (!isUsedAsSubproject) {
                 def variantOutput = variant.outputs.first()
-                def processResources = variantOutput.hasProperty(propProcessResourcesProvider) ?
+                def processResources = variantOutput.hasProperty(FlutterPluginConstants.PROP_PROCESS_RESOURCES_PROVIDER) ?
```

Comment:
Best I can tell this call makes no sense. See FlutterPluginUtils.findProcessResources for more discussion.